enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/configuration/IdentityVerificationConfig.java

```diff
@@ -51,11 +53,11 @@ public class IdentityVerificationConfig {
     @Value("${enrollment-server-onboarding.presence-check.cleanupEnabled:false}")
     private boolean presenceCheckCleanupEnabled;
 
-    @Value("${enrollment-server-onboarding.identity-verification.data-retention.hours:1}")
-    private int dataRetentionTime;
+    @Value("${enrollment-server-onboarding.identity-verification.data-retention.hours:PT1H}")
+    private Duration dataRetentionTime;
 
-    @Value("${enrollment-server-onboarding.onboarding-process.verification.expiration.seconds:300}")
-    private int verificationExpirationTime;
+    @Value("${enrollment-server-onboarding.onboarding-process.verification.expiration:PT1H}")
```

Comment:
Shouldn't be `5m` here?

Reply:
I think `5m` is too little. The user may need more than 5 minutes to get the ID card and driver's license.